server/src/services/server-info.service.ts

```diff
@@ -42,6 +47,35 @@ export class ServerInfoService {
     }
   }
 
+  async getAboutInfo(): Promise<ServerAboutResponseDto> {
+    const version = serverVersion.toString();
+    const { buildId, gitCommit, gitRef, imageTag } = getBuildMetadata();
+    const { nodejs, ffmpeg, libvips, exiftool, imagemagick } = await this.serverInfoRepository.getBuildVersions();
+
+    return {
+      version,
+      releaseUrl: `https://github.com/immich-app/immich/releases/tag/v${version}`,
+
+      buildId,
+      buildUrl: buildId ? `https://github.com/immich-app/immich/actions/runs/${buildId}` : '',
+
+      gitRef,
+      gitCommit,
+      gitCommitUrl: gitCommit ? `https://github.com/immich-app/immich/commit/${gitCommit}` : '',
+
+      imageTag,
+      imageUrl: imageTag
+        ? `https://github.com/immich-app/immich/pkgs/container/immich-server/234402559?tag=${imageTag}`
+        : '',
```

Comment:
Can URLs be changed? + it doesn't make sense to have an imageUrl for unofficial deployments like the Immich package in the AUR

Reply:
I could just make them configurable like everything else and pass them in as build args. That way you could link to your own releases.